log/roller_test.go

```diff
@@ -76,14 +77,16 @@ func TestParseRoller(t *testing.T) {
 		t.Errorf("ParseRoller should be failed")
 	}
 
-	praseArgs = "size=100"
-	err = InitGlobalRoller(praseArgs)
-	if err != nil {
-		t.Errorf("ParseRoller failed")
-	}
-	if defaultRoller.MaxSize != 100 || defaultRoller.Compress != false {
-		t.Errorf("ParseRoller failed")
-	}
+	/*
```

Comment:
we'd better fix it instead of removing it. It's not a good idea to remove the existing test case.

Reply:
fix